youtube_dl_gui/optionsframe.py

```diff
@@ -273,7 +273,8 @@ class GeneralTab(TabPanel):
         ('ko_KR', 'Korean'),
         ('pt_BR', 'Portuguese'),
         ('ru_RU', 'Russian'),
-        ('es_ES', 'Spanish')
+        ('es_ES', 'Spanish'),
+		('he-IL', 'Hebrew')
```

Comment:
* Sort alphabetically as the guide says
* Change the language code from 'he-IL' -> 'he_IL'
* Rename the 'he-IL' directory under the **locale** dir to 'he_IL' in order to match the new language code
* Follow the Python's conventions and align the Hebrew language to the other items (4-spaces)